src/features/Main/data/slice.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Why is it necessary? @AuraAlba

Reply:
Eslint give this error because the reducer modify directly the property of the state, the good practice is make copy of the object and modify it but redux improve that so we can use it, only with reducer of redux